presto-orc/src/main/java/com/facebook/presto/orc/reader/ReaderUtils.java

```diff
@@ -98,6 +98,24 @@ public static void packByteArrayAndOffsets(byte[] values, int[] offsets, int[] p
         }
     }
 
+    public static void packBytes(byte[] values, int[] positions, int positionCount)
+    {
+        for (int i = 0; i < positionCount; i++) {
+            values[i] = values[positions[i]];
```

Comment:
Does this function always moves values from the right side of the array to the left side?

Reply:
Yes, positions array is monotonically increasing.